app/components/user-page/course-progress-list.ts

```
@@ -15,7 +15,10 @@ export default class CourseProgressListComponent extends Component<Signature> {
   get courseParticipationGroups() {
     const participationsGroupedByCourse: CourseParticipationModel[][] = Object.values(
       groupBy(
-        this.args.user.courseParticipations.filter((participation) => !participation.course.releaseStatusIsDeprecated),
+        this.args.user.courseParticipations.filter((participation) => 
+          !participation.course.releaseStatusIsDeprecated && 
+          participation.course.visibility !== 'private'
```

ESLint

[error] Replace `(participation)·=>·⏎··········!participation.course.releaseStatusIsDeprecated·&&·⏎··········participation.course.visibility·!==·'private'` with `⏎··········(participation)·=>·!participation.course.releaseStatusIsDeprecated·&&·participation.course.visibility·!==·'private',`

(prettier/prettier)